src/websocket.js

```diff
@@ -393,6 +393,12 @@ Strophe.Websocket.prototype = {
      * (Object) error - The websocket error.
      */
     _onError: function(error) {
+        // Workaround
+        // https://code.google.com/p/chromium/issues/detail?id=372776
+        if (this.socket.readyState === 0) {
+          return;
+        }
+
```

Comment:
for a proper fix maybe we can add disconnected property and ignore error if disconnected ==== true ?